pkg/k8s/k8s.go

```diff
@@ -329,3 +330,27 @@ func GetCronjobNameWithSchedule(c *clients.Clients, namespace, schedule string)
 	}
 	return name
 }
+
+func AssertPrunerCronjobWithContainer(c *clients.Clients, namespace, num string) {
+	log.Printf("Verifying if the cronjob with prefix tekton-resource-pruner in namespace %v contains %v number of containers", namespace, num)
+	cronJobs, err := c.KubeClient.Kube.BatchV1().CronJobs(namespace).List(c.Ctx, metav1.ListOptions{})
+	if err != nil {
+		testsuit.T.Errorf("Error while getting cronjobs %v", err)
+	}
+	jobFound := false
+	for _, cr := range cronJobs.Items {
+		if strings.Contains(cr.Name, "tekton-resource-pruner") {
+			jobFound = true
+			containers := cr.Spec.JobTemplate.Spec.Template.Spec.Containers
+			numInt, _ := strconv.Atoi(num)
+			if len(containers) != numInt {
+				testsuit.T.Errorf("Expected: %v number of containers in cornjob spec, Actual: %v number of containers in cronjob spec")
```

Comment:
I suggest to use messages that read more naturally, e.g.
```
Expected: %v containers in cronjob spec, Actual: %v containers in cronjob spec
```

(btw, a typo in cornjob)